modules/andes-core/broker/src/main/java/org/wso2/andes/kernel/distruptor/inbound/InboundKernelOpsEvent.java

```diff
@@ -59,7 +63,7 @@ public void updateState() throws AndesException {
                 stopMessageExpirationWorker();
                 break;
             case SHUTDOWN_MESSAGING_ENGINE_EVENT:
-                shutdownMessagingEngine();
+                gracefulShutdown();
```

Comment:
shouldn't it be like : gracefullyShutdown()...!! (just a optional very trivial change)